### PR TITLE
Fix error in image about aad resource-server.

### PR DESCRIPTION
**Pull request by chenrujun**
Fix error in image about aad resource-server.

### CODE REVIEW

File: sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio [hunk 1]
```diff
@@ -1 +1 @@
...gw9bgrWxtHQglVA+lDxsnjj4eTmeXiROo60fjBHRh92ZxYn+cOLHOtbZp/WNxQiz2x+bm0+7JiZcW/Lyf3NzjXsGANdtin9cusltZclpJW5as5lwTkkj7fceM7WUhhtaMNBM8NyrdP9Y7T0JZ3xy4hWCyk+rz3r7eq6GaCRc8BOVa+vngifd9ETqKI04Ym2Yt14WEfPRWOGeNGpuQ6wPQ57mp+tlNhQzRADOpo16JHVFrtvTwooB6WNEzpBTta9NkJB3cfRn7rXUObUuZP2gUlLyR76By2dLSr/DigUIZX1EU+oiJsgh5Hk7TrOZ/xirtC/yv0ZyfOOCIPxSIMtPjnpORhohsni+ioRxYhb6fx4DgclSRvMpIx73yxYhUOKU8nshVtfoJoItreqBvu1C6xSud2DH7A7sJbEsHWSxSzLQ2IZ2HbmecquDeYbamohZORWrlTMSPhoghBdo2enPCGFllNOIPxblOUFzCfaJ15uDDEjlH9HYheyz4h7dzrnKtE9L2URfQ34eDdAc2wIOu1Xfrv7/HQdCyz1F7gJIu5SeO4ifDIvJ5tDIaeixzFYrzjaTa2NCG9aMUWdSG89G0+0x1imfNI+I9fygnaRbx+mBwEn5H3UJRdMC/LpUOVMLgGEdMWD9BcQNI+8+1uDoZzZH3HGR1Tc/L6+yhMGgw/yVLtHwfwBBUKhq67f6a4QQEdL28phfTdT3ZZQOFVt4KxH+9WJ+/Tr6FXK641VnXJJFIWqNy47XzgTl07Nms63w9HQ/d0guLkZhkJ52THfGMQ7iltzxB8lC94JVxWy949QtcsXS6RnmP9u+ud9+JMaDd4n9oN2rblsKT00pbrW70AThcMl+o1DVbJ2G+7IvbVObDRmn8NqWuoV4STkU7L3X5KXgVpml+UThHaZbv6gkuYz+P1zxpzmWCOhKRIbNbRUoinHYGyHsUyJwZvTAO7iQNgEp0jRdCotunl9FFoP9cZbQL2mW0229XSRcppLtt0nVR/7rECGq5EdTSowEOZ0glPcL/06NkEMd+t/TY+SVHvTa9K6rhg58X+LuypuW7zVeztRSdTiZdUDed4Z2+Gblmv3nvYnYgBo1iUh004wJ3eNSm5rfv8TcXBDMY956MrYs7Phv9FFFt/FejurzVlS5iFB8MLx/VvFt9+83TRPUBHU7/Ag==</diagram></mxfile>
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio:1]: image

File: sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio [hunk 2]
```diff
@@ -1 +1 @@
...
...f9MSTrW0ZDWPbHcYGHbYGb2Vr60AooXoofdg4cfTxcDq7TJxAXT8aJ6ALuzeLE/vjxIl1rrVN6x+LE2KxPzY3n3ZPTry04Of95OYe94oMWLMt9jl3kd3KktNK2rJkNeeakETa7ztmbC+JGFoz0izw3Kh0/1jvPAllfXPgFoLJTqrPe/t6r4ZqJlzwEJRr6eeDJ973RegojnjC2DS5XBcS8tFb4Zy11NiEXB+APq9N1c9uKmSIBphJHXUmdkSt2dLDSQH1sKJnSCna16bJSDq4+zL2W+sc2pYyf9AglLyR76By2dLSr/DigZIyvqIo9BETtAh5Hk7TjPM/YzXtC/yv0ZwfOeCIPxQImulxz8mShohsXi+ioRxYhb6fx4DI5ahK8mpGOu6VL0akklPK44lcVaufALpyTQ/0bRdKt3ilEztmf2A3gW3pIItFipnWTkjnodsZpyq4d5itqeDCqSitPBPxoyFiSIG2jd6cMEZWWRrxh+JcJ1Jcwn2idebgwxI5R/R2IXss8g9v57nKtU4o20ddQH+fHKQ7sAEedK2+W//9vRwELfsctQdS0qX8xFH8ZFhEPo9WRkOPZa5Ccb6RVBsb2rB+lCKL2nA+mnafqU7xrHlEvOcP5SRNEq8PBifhd9QtFEUH/OtS5UBNGBzjiAnrJyhuAGn/uRZXJ6M58p6DjNf0vJxnD4VBg/kvWaHl+wCGSKWiodvurxlOQEDXyzm9mK7ryS4bKLTyViD+9WJ9/jr5FnK54lZnXZNEomiNyo3Xzgfm0LFns67z9XQ8dEsvLEZikp10TnbEMw7hlt7yBJmH6oRXxm2d8OoXuGLpdI3yHu3fzXffKWNAu5X/od3gti2FJ5eVtlrd6ANwmDJfiOqarZMwX/bFbSrzYYMavw3VNdRLwqlo51SXn4JXYZrmF4VzlGb1rl7gYlnVPE570n5nLLwHF+ZJ0Avj4E5GPKhE13ghJLp9OmMuYvrnYswuaDNmt98mRBfhzN026bqTf10NBLUyCGqV0ACHi6FSCeH/lVAmC8d+t0rY+dFGvSG9K4jvwS8J/F1Z0/Ld5qvZWopOTyZdUDed4Z0+D7lmv3nFYnYgBo1iUh004wLXddSm5rfv8TcXBDMY956MrYs7vhD9FFFt/FejurzAlS5iFN8GLx/VvFt95s3LRPWxHE7/Ag==</diagram></mxfile>
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio:1]: image

Comment (backwind1233) [reply sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio:1]: why do I feel that the image is not ver clear @chenrujun

File: sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio [hunk 3]
```diff
@@ -1 +1 @@
...
...ca7veT4sT5ok/Hy4/bO+G8cqDH3fTy2vcKzJgzbY4ktxFNytLnlXSliWrMReMJdp+37EQO03E0EawZoGXRuW723rjTinru4OgEEy3Wn3e2tVbNVQz4UKGoJ4LPB889b1PQscxlQnjocnlupDQr35WzllLjU3IwcDuy9pU/fymQoH4Eguto87EDqh1W3okKeAhNvSMOEe72jAdSXtXX8Z+a559yzLGDxqEUj7kK6hctrT0C7x4YKSMr4iSCAlFi1AY4jTNOP89jq2JY43q/GmuzBFSRNZqDCX3OOdXf20UqR6jTVQ1LGfS6CRKb9Hod4bSBZrLzYzUQ8lSMViKF2qwShmyENGRFq9JFOXBpYoEqqqHmeoOu/uToW4kq3Lfo2e16luLriTWs/t+ALW/vTA6YJOEg5YCtlikWFjtNPc8n+mMftNlrrDYcMWwU1WwZX6TG04kkIFqG7o5E4Kts+QUjdRuUSXORHpaaycjuzVsQ9XaEnFbZDX5nGfAwDuCDBzEH7xNZgND2AAPBl4/qP/+WWaDnv8ctXsS3an8ZGj4yajIJzJUBSehyFyF43whqZFg2KLWnfem3Tu1YzxrTll4/66cpLk1AIPBUfgddAtD0R7/OlWRMROGxJiqXJ4mKG4AWVSEOQrvlxlb6oU5ex8pgy7nv2XlW9USR+VR9QD84PcMJ1tB18t3Cmo4AMk266iKkL9U/0Exv/ycfAm53HCrZx2+UFWvxuXCa7sOdzT0Z7OuXfvZZBSUXlj0xCzbPx3tiM/Y2nug5Qk6D9VptI7bOo0GJzi46XSN8nTu382i3yhjQL+V/6HfYMwthUeXlbZa4PRtez8RPxGBdlv7azntk8s0xsMG4X4dAu2YR4/jDaGqrkm/WzFOBMGq5DXIclnhDNKchsl+0swZxU3aXK+WX1ZYJVOslpLzd7m1X0uiTVj8sxi2zK4hiZdXOpXYlegim2sK/ON5eJEsfi0eDlo8vHXgcBIe3m2OrtuDl9VVu1Za7Vp1dez9BdaorvD/6qoT0NB/s+raeb1knuVeFWR6752H/FbRtHy3+Wq21qIii4TSgtm54d480gV10xne6CIrcPvNwyC3AzHoFIPqoDknOFjkPne/fY+/BfZyBuPenfMY4I67rF8iqp3/alSXR83aRZziFvP0US2b1YV0XiaqW3149jc=</diagram></mxfile>
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio:1]: image

File: sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio (hunk 1 above)
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio:1]: image

File: sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio (hunk 3 above)
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio:1]: image

**Comment by chenrujun** [comment]
Hi, @backwind1233 , @zhichengliu12581 , @yiliuTo .
Please help to review this PR.

File: sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio [hunk 4]
```diff
@@ -1 +1 @@
...
...xf1qcUJf/ubn9sHvqRwvX/rgf396jTp4BK7pFoeAuqllq8qpEG5osx9xQGiv9fUec7xURg2tO6wVeKJXtH6uNJyms6/T8HBjvlPista+2KlZNwZkIQTUXON948nufNR1DRCSMTZ3LtVlCvfpZOmclNdZNDnpmV9Sm8ufVBXLI5ogrGVUmdkSs05AjSAELkCZnwBjcV4apSDq4+iL2G/McWpY2vlcjlOIhW0HpsoWmX+DFPS1lfIUEh5BLWgSDACVJyvmXKDJGljGo8qepVEdAIF7JMQQvUcav/lpLUj2E67BsGNao1oml3LzRbQ2lGzgVmxkhh+C5ZLAEzeRgmTJEISIDBa9wGGbBJYsELKuHnuqOu/uzoa4lq2Lfo2Y1qluLtiTWMbuebyt/e2F02HUSDhoC6GyWIG4009x5PtMa/brL3CG+ZpJhJ7Jgi/wmNpyQQ82qTdNNKed0lSancCB3izJxxsLTGjsZ0a3M1petHeaPeVYTz1kG9N0TyMBR+4O3yWygb9eMZ/tu16/+/llms13vHLEHEt2l/KSv+ckgzyciVDnDAU9dhaFsIYmWYOis0p31Ju07tVM8a0posHxXTlLfGoBe7yT7HXULTdAB/7pUkdEThrAxkbk8iWFUM2ReEaYwWM5TttQJMvY+kAqdT39Ly7esJZbMo/IBeP7vqZ1MabpOtlOQwwGId2lHWYS8ufwP8vnF52RLyHDNrc46fCGyXg2LhVd2Hc6g700mbbv2q9HAL7ww74loun862RHP2Nq7oOEJKg9VabSK2yqNBhc4uGl1jeJ07t/Not8oY9heI//bXo0xNwSeXFaaYoHVNc3DRPxCBNpp7K/FtM8uUxtv1wj36xBoSz96HK4xkXVN+N2CMswxkiWvRpaLCqeR5iSID5NmRgmq0+ZqtfyyQDKZIrmUjL+Lrf1KEG1Mo5/FsEV2DXA0v1OpxCyhm3SuMfBO5+F5svi1eDho8PDGgcNFeHi7OtpuD15WV81KaTUr1dUyDxdYrbra/1dXlYDy27c3qK6t10v6We5dTqYP3nmIb+V1zberr6JrBeVZJBAaTM8ND+aRNlPXneGNLrJ8p1s/DHJaLGZb+aCq0awLHCwyjznfvkfffHM+saPOk7X1Uctd1i8R1dZ/NaqLo2blIlZ+i3n5qBbN8kI6KxPlrb599Tc=</diagram></mxfile>
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio:1]: image

File: sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio [hunk 5]
```diff
@@ -1 +1 @@
...
...1+Z+b2w+7p360cO2P+/HtPerkGVDTLQoFd1HNQpNXBVrRZDHmhtJY6e874nyviBhcc1ou8EKpbP+oN56ksK7T83NgvFPis9Zeb2lWTcGZCEE1FzjfePJ7nzUdQ0QkjE2ZyzVZQr36WTqnlhrLJgc9sytqU/HjlQVyyOaIKxk6Ezsi1qnIEaSABagmZ8AY3GvDVCS1rv4Q+5V52pZVG98rEUrxkK2gcNmDpl/gxb1ayvgKCQ4hl7QIBgFKkpTzL5FkUFOpgIBAvEo0HkXwEmXM6q+1pNNDuA6LhmGNSp1YSmzrjKaz9k60i4tGtzH8buBUbIDECgieS9ZL0EwOlmlGFC8yUPAKh2EWkLKwwKLi1NPj8RB5Nj3UEtxhr6RmNfTtSFPi65hdz7eVj74wouwycQcVAXQ2SxA3qqnxPD9rzBh1N7tDfM2kTyWyyIucKDapkMOaVaumm1LO6SpNaOFA7jBlso2Fd1Z2P6Jbma2fug7mj3kmFM9Z1vTdEwjEUfuDt8mGoG+XjGf7btfXf/5ZNrRd7xyxLcnxUn7Sr/nJIM9BIlQ5wwFPXYWhbCGJMbKMgb6pozOtO+tNmnd3p3jWlNBg+a6cpLydAL3eSfY76hY1QS3+danCVE8YwsZE5vIkhlHJkHlFmMJgOU8ZVifIGP9AKnQ+/S0t+bKWWDKPygfg+b+ndjKl6TrZ7kIOByDepR1FEfLm8i/I5xefky0hw2tuddaBDZH1anhYuLZTcQZ9bzJp2ulfjQb+wQvznoime66THfGM4wAXVDxB5SGdequ41ak3uMBhT6NrHE70/t3M+40yhu1V8r/tlVh2ReDJZaUqFlhd02wn7xci3U5lTy6mfXaZtfF2iaS/Dum26seVwzUmsq4Jv1tQhjlGyel0OwnidtLMKEFJiTbr1fLLIuX1SC4l4/wxYitB0TGNfhbDFtk1wNH8TqUSs4Bu0rnGwDudh+fJ4tfi4aDCwyuHFBfh4c3qaLpxeFldNbXSamrV1TLbC2ytutr/V1eVgPIbuzeoro1XUvXz37ucTLfek4hv5WXNN6tP07WC8iwSCA2mZ42teaTJ1GVneKPLL9/plg+QnAaL2VY+SDeadYHDSOYx59v36Jtvzid21Hmytj5quP/6JaLa+q9G9eF4WrmIld98Xj6qRbO4xM7KRPGfAPbV3w==</diagram></mxfile>
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio:1]: image

Comment (backwind1233) [reply sdk/spring/azure-spring-boot-starter-active-directory/resource/source-file/resource-server.drawio:1]: Look good for me @chenrujun

**Review by backwind1233 — APPROVED** [review]
LGTM

**Comment by zhichengliu12581** [comment]
LGTM

**Pull request merged**